pkg/dashboard/manifest.json.in

```diff
@@ -7,7 +7,7 @@
     "dashboard": {
         "index": {
             "label": "Dashboard",
-            "order": 0,
+            "order": 10,
```

Comment:
On my system, with the registry installed, Cockpit defaults to `https://[ip]:9090/kubernetes/registry` after login. Before, it defaulted to the dashboard. I guess that's related to this line.